pyke/lightcurve.py

```diff
@@ -118,6 +118,15 @@ def fold(self, period, phase=0.):
         sorted_args = np.argsort(fold_time)
         return LightCurve(fold_time[sorted_args], self.flux[sorted_args])
 
+    def bin(self, binsize):
+        flux_err = None
+        if self.flux_err is not None:
+            flux_err = np.nanmean(self.self.flux_err[:-q or None].reshape(-1, N), axis=-1)
```

Comment:
To do correct uncertainty propagation, should this be the square root of the sum of squares?